### DISCUSSION TITLE
Send enterprise contributions to github.com profile

### DISCUSSION THREAD

**Discussion by Ayush21298** [community/community#38235] [Other Feature Feedback, Questions, & Ideas]
When using "Send my contribution counts to GitHub.com" under "GitHub Connect", is there any way to send all contribution counts (by default it's limited to past 90 days) ?

**Comment by ossanna16** [comment]
Hi there @Ayush21298 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
